libdevcrypto/hsm/HSMSignature.cpp

```diff
@@ -81,8 +95,12 @@ std::shared_ptr<crypto::Signature> dev::crypto::SDFSM2Sign(
         key, hsm::SM2, (const unsigned char*)hashResult, 32, signature.data(), &signLen);
     if (code != SDR_OK)
     {
-        throw provider.GetErrorMessage(code);
+        CRYPTO_LOG(ERROR) << "[HSMSignature::sign] ERROR of sign"
+                          << LOG_KV("error", provider.GetErrorMessage(code));
+        return nullptr;
     }
+    // cout<<"hash "<< toHex(hashResult)<<endl;
+    cout << "Signature = " << toHex(signature.data(), signature.data() + 64, "") << endl;
```

Comment:
Do not use std::cout in your code，use LOG instead

Reply:
Sure